chirho/counterfactual/ops.py

```diff
@@ -23,10 +25,61 @@ def split(obs: T, acts: Tuple[Intervention[T], ...], **kwargs) -> T:
     return scatter(act_values, event_dim=kwargs.get("event_dim", 0))
 
 
+@pyro.poutine.runtime.effectful(type="undo_split")
+def undo_split(
```

Comment:
Can you move `undo_split` to a new file `chirho/handlers/counterfactual/explanation.py` that we will use for subsequent PRs in this series?

Reply:
I take it you meant `chirho/counterfactual/handlers/explanation.py` here, which matches the existing folder structure?

Reply:
Sorry, yes, that's right

Reply:
done.